ingestion_tools/scripts/data_validation/tests/test_neuroglancer.py

```diff
@@ -0,0 +1,19 @@
+from typing import Dict
+
+import allure
+import pytest
+from tests.test_deposition import HelperTestDeposition
+
+from common.fs import FileSystemApi
+
+
+@pytest.mark.tomogram
+@pytest.mark.parametrize("run_name, voxel_spacing", pytest.run_spacing_combinations, scope="session")
+class TestNeuroglancer:
+    @allure.title("Neuroglancer: sanity check neuroglancer config file.")
+    def test_metadata(self, neuroglancer_config: Dict):
+        assert len(neuroglancer_config) > 0
```

Comment:
Can we update this test to test there is at least one layer in the config? 
```suggestion
        assert "layers" in  neuroglancer_config and  len(neuroglancer_config["layers"]) > 0
```